src/aat/resources/features/F-1009 - Submit Case Creation Handle Case Links/CaseLinkVerifications/F-1009-VerifyCaseLinksNotInserted.td.json

```diff
@@ -0,0 +1,41 @@
+{
+  "title": "should retrieve empty list of case links for a specified case reference",
+  "_guid_": "F-1009-VerifyCaseLinksNotInserted",
+
+  "productName": "CCD Data Store",
+  "operationName": "Retrieve case links for an existing case reference",
+
+  "method": "GET",
+  "uri": "/testing-support/case-link/{cid}",
```

Comment:
Can we have a **common TD file** to cover the basics of this new API signature: see

* F-1008-VerifyCaseLinks
* F-1009-VerifyCaseLinks
* F-1009-VerifyCaseLinksNotInserted
* F-1009-VerifyCaseLinks_V2
* F-1009-VerifyCitizenCaseLinks
* F-1009-VerifyMultipleCaseLinks
* F-1009-VerifyMultipleCitizenCaseLinks


and see also the following in hmcts/ccd-data-store-api#1719:

* F-1008_VerifyCaseLinksBase

Reply:
Done [c129a9a](https://github.com/hmcts/ccd-data-store-api/pull/1872/commits/c129a9a6a6fdff4226e65e0b027691d3cd49352f)